packages/instant/src/components/sliding_error.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+
+import { ColorOption } from '../style/theme';
+
+import { SlideUpAndDownAnimationComponent } from './animations/slide_up_and_down_animation';
+
+import { Container, Text } from './ui';
+
+export interface ErrorProps {
+    icon: string;
```

Comment:
For right now the only errors we have in the UI are emoji based, so using string here instead of needing to support JSX.Elements